synapse/api/auth.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, hs: "HomeServer"):
         self.clock = hs.get_clock()
         self.store = hs.get_datastore()
         self.state = hs.get_state_handler()
+        self._account_validity_handler = hs.get_account_validity_handler()
```

Comment:
Also worth noting that this isn't a new issue since the line just above gets the state handler (and the account validity handler was already called in this file before this change, the PR is merely moving where we get it from the `hs` object).

Reply:
I'm not seeing that, tbh. But it doesn't really matter.

Reply:
Oh yeah I was mixing up things, mb